script.js

```diff
@@ -29,38 +40,25 @@ const addItemsFeatures = (newItem) => {
 
 userInput.addEventListener("keydown", (key) => {
   if (key.keyCode === 13) {
-    const item = userInput.value.replace(/^\w/, (c) => c.toUpperCase());
-    const VALID_ITEM = /^[^0-9]{2,}$/;
-    if (VALID_ITEM.test(item)) {
-      itemsCounter++;
-      createNewItem(item);
-      userInput.value = "";
-      userInput.focus();
-    }
+    enterItem();
   }
 });
 
-document.querySelector(".insertItemBtn").addEventListener("click", function () {
-  const item = userInput.value.replace(/^\w/, (c) => c.toUpperCase());
-  const VALID_ITEM = /^[^0-9]{2,}$/;
-  if (VALID_ITEM.test(item)) {
-    itemsCounter++;
-    createNewItem(item);
-    userInput.value = "";
-    userInput.focus();
-  }
+document.querySelector(".insertItemBtn").addEventListener("click", () => {
+  enterItem();
 });
 
-document.querySelector(".orderAZ").addEventListener("click", function () {
+document.querySelector(".orderAZ").addEventListener("click",  (btn) => {
```

Comment:
the name of the event callback parameter is not really matter from logic perspective ,  but from semantic perspective we should prefer using hint that actually mean what it is... (e.g: `evt` , `event` , `e`), using `btn` as a name is actually incorrect. (same idea with `key`)

Reply:
got it! fixed